package.json

```diff
@@ -10,11 +10,15 @@
   "dependencies": {
     "@storybook/addon-docs": "^6.0.0-beta.26",
     "@storybook/preset-create-react-app": "2.1.1",
+    "@types/moment": "^2.13.0",
+    "@types/react-dates": "^21.8.0",
     "array-move": "3.0.1",
     "babel-plugin-inline-json-import": "^0.3.2",
     "lodash.merge": "^4.6.2",
+    "moment": "^2.29.1",
```

Comment:
moment is so huge... but i guess there is no alternative here?

Reply:
That's the issue 😭😭😭😭 I didn't payed attention that airbnb used this lib under the hood 😢 It could be a blocker to use this component (it exists so many) should we continue with this one for now?

Reply:
there are ways to minimize the impact with webpack common junks or a cdn or similar. So i don't see this as a blocker and actually there is nothing as good as moment at the moment. So keep going with it